SuperBuild/External_python-scipy.cmake

```diff
@@ -31,16 +31,16 @@ if(NOT Slicer_USE_SYSTEM_${proj})
   file(WRITE ${requirements_file} [===[
   # [scipy]
   # Hashes correspond to the following packages:
-  #  - scipy-1.9.2-cp39-cp39-macosx_10_9_x86_64.whl
-  #  - scipy-1.9.2-cp39-cp39-macosx_12_0_arm64.whl
-  #  - scipy-1.9.2-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
-  #  - scipy-1.9.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
-  #  - scipy-1.9.2-cp39-cp39-win_amd64.whl
-  scipy==1.9.2 --hash=sha256:1e3b23a82867018cd26255dc951789a7c567921622073e1113755866f1eae928 \
-               --hash=sha256:82e8bfb352aa9dce9a0ffe81f4c369a2c87c85533519441686f59f21d8c09697 \
-               --hash=sha256:61b95283529712101bfb7c87faf94cb86ed9e64de079509edfe107e5cfa55733 \
-               --hash=sha256:8c8c29703202c39d699b0d6b164bde5501c212005f20abf46ae322b9307c8a41 \
-               --hash=sha256:7b2608b3141c257d01ae772e23b3de9e04d27344e6b68a890883795229cb7191
+  #  - scipy-1.10.0-cp39-cp39-macosx_10_15_x86_64.whl
```

Comment:
@jcfr take note that the whl has changed from `scipy-1.9.2-cp39-cp39-macosx_10_9_x86_64` to `scipy-1.10.0-cp39-cp39-macosx_10_15_x86_64.whl` here.

Reply:
Re: scipy
What do you think of keep scipy labelled as 10.9 so that works with 10.13 ?

That way, we can backport to 5.2.2.

Once, we have updated the nightly to the recent macos, we will then update scipy to the latest.

Reply:
@jcfr I've update this PR to remove the `scipy` update. I will issue it once the factory machine switches to computron with the latest macOS.